homeassistant/components/plex/media_player.py

```diff
@@ -258,24 +258,24 @@ def username(self):
     def update_state(self, state):
         """Set the state of the device, handle session termination."""
         if state == "playing":
-            self._attr_state = STATE_PLAYING
+            self._attr_state = MediaPlayerState.PLAYING
         elif state == "paused":
-            self._attr_state = STATE_PAUSED
+            self._attr_state = MediaPlayerState.PAUSED
         elif state == "stopped":
             self.session = None
             self.force_idle()
         else:
-            self._attr_state = STATE_IDLE
+            self._attr_state = MediaPlayerState.IDLE
 
     @property
     def _is_player_active(self):
         """Report if the client is playing media."""
-        return self.state in (STATE_PLAYING, STATE_PAUSED)
+        return self.state in {MediaPlayerState.PLAYING, MediaPlayerState.PAUSED}
 
     @property
     def _active_media_plexapi_type(self):
         """Get the active media type required by PlexAPI commands."""
-        if self.media_content_type is MEDIA_TYPE_MUSIC:
+        if self.media_content_type == MediaType.MUSIC:
```

Comment:
Slight change of logic here, to ensure string matching. If it is safe to use `is` then I can adjust but it felt safer here.

Reply:
`is` won't work here since it checks identity. In (at least) Python 3.10 `is` would even raise a `SyntaxWarning`.
```
SyntaxWarning: "is" with a literal. Did you mean "=="?
```